waku/v2/protocol/filter/filter_health_check.go

```diff
@@ -0,0 +1,48 @@
+package filter
+
+import (
+	"context"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"go.uber.org/zap"
+)
+
+func (wf *WakuFilterLightNode) PingPeers() {
+	//Send a ping to all the peers and report their status to corresponding subscriptions
+	// Alive or not or set state of subcription??
+	for _, peer := range wf.subscriptions.GetSubscribedPeers() {
+		go wf.PingPeer(peer)
+	}
+}
+
+func (wf *WakuFilterLightNode) PingPeer(peer peer.ID) {
+	ctxWithTimeout, cancel := context.WithTimeout(wf.CommonService.Context(), wf.peerPingInterval)
+	defer cancel()
+	err := wf.Ping(ctxWithTimeout, peer)
+	if err != nil {
+		wf.log.Info("Filter ping failed towards peer", zap.Stringer("peer", peer), zap.Error(err))
```

Comment:
Did not make it into an error, because it could just be routine flow where ping towards peer fails. 
If a user has bad connectivity, this may flood the logs.